src/tests/ExtendedDateFilters.ts

```diff
@@ -0,0 +1,60 @@
+// (C) 2019 GoodData Corporation
+import { ExtendedDateFilters } from '../ExtendedDateFilters';
+
+describe('ExtendedDateFilters', () => {
+    const absoluteFilter: ExtendedDateFilters.IAbsoluteDateFilterForm = {
+        from: '2000-01-01',
+        localIdentifier: 'foo',
+        to: '2020-01-01',
+        type: 'absoluteForm'
+    };
+
+    const relativeFilter: ExtendedDateFilters.IRelativeDateFilterPreset = {
+        from: -2,
+        granularity: 'GDC.time.date',
+        localIdentifier: 'foo',
+        name: 'bar',
+        to: 0,
+        type: 'relativePreset',
+        visible: true
+    };
+
+    const allTimeFilter: ExtendedDateFilters.IAllTimeDateFilter = {
+        localIdentifier: 'foo',
+        type: 'allTime'
+    };
+
+    describe('isAbsoluteDateFilterOption', () => {
+        it.each([
```

Comment:
could be merged with true

Reply:
It could, but then we would have 4 `false`s and only 1 `true` which seems not better